src/services/fluence-aurora-connector/modules/fluence_aurora_connector/src/jsonrpc/deal_changed.rs

```diff
@@ -0,0 +1,128 @@
+use marine_rs_sdk::marine;
+
+use crate::chain::chain_data::ChainData;
+use crate::chain::deal_changed::{DealChanged, DealChangedData};
+use crate::hex::{hex_to_int, int_to_hex};
+use crate::jsonrpc::get_logs::GetLogsReq;
+use crate::jsonrpc::JsonRpcReq;
+
+const DEFAULT_BLOCK_RANGE: u64 = 9999;
+
+#[derive(Debug)]
+#[marine]
+/// RENAME: why is it called `DealUpate`?
```

Comment:
`DealUpdate` is used to find an update for the deal `deal_info.deal_id` from the block number `from_block` on the chain.

I called it like that based on the usage purpose. This data structure doesn't describe anything else. Mb `DealUpdateReq` or smth like that can be more clear?

Reply:
it looks like it functions as arguments for request that retrieves a log of deal updates from chain. Gotta think about the name, maybe keep it, maybe come up with an other.

Reply:
I was thinking `DealChangesRequest`